### PR TITLE
Implementation of inital version

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		cli.StringFlag{
+			Name:   "language",
+			Usage:  "language of coverage",
+			Value:  "go",
+			EnvVar: "PLUGIN_LANGUAGE",
+		},
+		cli.StringFlag{
+			Name:   "commit.sha",
+			Usage:  "git commit sha",
+			EnvVar: "DRONE_COMMIT_SHA",
```
Comment (tonglil) [inline main.go:43]: Would it also be helpful to add a generic `PLUGIN_COMMIT_SHA`?

**Review by tonglil — APPROVED** [review]
Not familiar with appveyor to review building for windows.

LGTM though packages.go seems a little complex w/o any tests to check outputs, it could be brittle to future changes.

File: main.go (hunk 1 above)
Comment (tboerger) [reply main.go:43]: This gets always set by drone, so IMHO this doesn't make sense.

**Comment by tboerger** [comment]
> LGTM though packages.go seems a little complex w/o any tests to check outputs, it could be brittle to future changes.

I can try to provide proper tests later on.

**Pull request merged**